examples/monolithic_build_multilevel_native/mlkem_native_all.c

```diff
@@ -4,7 +4,7 @@
  */
 
 #define MLK_MULTILEVEL_BUILD
-#define MLK_EXTERNAL_API static
+#define MLK_EXTERNAL_API_QUALIFIER static
```

Comment:
Why do we need 
`#define MLK_EXTERNAL_API_QUALIFIER static` in both `mlkem_native_all.c` and `multilevel_config.h`? Isn't it sufficient in one place?

Reply:
Good spot!

Reply:
Fixed